tools/generate_config_gypi.py

```diff
@@ -5,7 +5,7 @@
 def main(args):
   out = args[0]
   with open(out, 'w') as f:
-    f.write("{'variables':{}}\n")
+    f.write("{'variables':['built_with_electron': 1]}\n")
```

Comment:
errr, should this be `{}`? this doesn't look like it'll parse

Reply:
Yep, should be `f.write("{'variables':{'built_with_electron': 1}}\n")`

Reply:
@codebytere did this on my suggestion which was in part based on https://gyp.gsrc.io/docs/InputFormatReference.md#Variables-and-Conditionals